tests/framework/cli/test_cli.py

```diff
@@ -77,7 +77,7 @@ def test_cli(self):
         """Run `kedro` without arguments."""
         result = CliRunner().invoke(cli, [])
 
-        assert result.exit_code == 0
+        assert result.exit_code in (0, 2)
```

Comment:
```suggestion
        assert result.exit_code == 2
```
Since we are pinning to >=8.2 we don't need the option